sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,22 +1,40 @@
 # Release History
 
-## 4.19.1 (Unreleased)
+## 4.20.0 (Unreleased)
 
 ### Features Added
-
 ### Breaking Changes
+### Bugs Fixed
+### Other Changes
+
+## 4.20.0b2 (2024-04-25)
+
+This is a Beta of the EventGridClient
 
 ### Bugs Fixed
 
-### Other Changes
+- Fixed serialization issues with CloudEvent and CNCF Cloud Event
+
+## 4.20.0b1 (2024-04-11)
+
+### Features Added
+
+- This is a Beta of the EventGridClient
+  - EventGridClient `send` can be used for both Event Grid Namespace Resources and Event Grid Basic Resources.
+    - Added a kwarg `level` in the EventGridClient constructor. The default value is `Standard` which creates a client for an Event Grid Namespace Resource.
+
+### Breaking Changes
+
+- Removed the `AcknowledgeOptions`,`ReleaseOptions`, `RejectOptions`, and `RenewLockOptions` models. `lock_tokens` can now be specified as a `kwarg` on the operation.
```

Comment:
maybe a link out to the sample showing the new usage?

Reply:
this is from the old beta can add to the ga changelog update